packages/trader/src/App/Components/Elements/VerticalTabs/vertical-tab-header.jsx

```diff
@@ -54,6 +55,9 @@ const VerticalTabHeader = ({ children, className, is_routed, item, onChange, sel
                 <HeaderIcon icon={item.icon} is_active={is_active} />
                 <Header text={label} />
                 {children}
+                {item.is_new &&
```

Comment:
Can we make this more generalized? 

So we can pass other `text` and `style` as props instead of only `NEW` with a `red` bg? 

Since `VerticalTabs` will eventually be moved to `components` we should handle `localization` from outside the component before it's passed as a `prop`.

Reply:
Okay 👍